### PR TITLE
Adding Project Update Reporting Years facet to project and field study browsers

### PR DESCRIPTION
**Pull request by zednis**

### CODE REVIEW

**Comment by zednis** [comment]
Code review for update that would add Project Update > Reporting Year facet to field studies and project browsers.  I don't see a user for Han on github so I will email him a link to the pull request / code review.

@mrpatrickwest

**Comment by nahgnaw** [comment]
My (Han) username is nahgnaw.

**Comment by zednis** [comment]
Adding @nahgnaw

**Comment by zednis** [comment]
Note, this commit includes more than just the changes for the project update reporting years facet - it turns out the initial commit of the repository had a problem with the opensearch/services folder that caused all its subfolders and files to not be added to the repository.  They have been added + updated for the new facet in this commit.

**Comment by zednis** [comment]
@mrpatrickwest I commented out the inclusion of the project update section in the result set.  The supporting code is still all there, I am just not calling it to add the project update section to the result set HTML.

The reporting year facet is in this branch.

I would like to merge this pull request into the master, push the reporting years facet to PROD, and close #dco303

File: ontologies/dco-s2s.ttl [hunk 1]
```diff
@@ -123,6 +116,17 @@
     s2s:hasOutput core:LabelIdContextCountJsonArray ;
     a s2s:InputValuesInterface, owl:Thing .
 
+:ReportingYears
```
Comment (zednis) [inline ontologies/dco-s2s.ttl]: this is stupid code!  Why are you a stupid head!

**Comment by mrpatrickwest** [comment]
:shipit:

**Comment by zednis** [comment]
sweet!  merging branch with master, closing pull request, branch, and https://orion.tw.rpi.edu/bloodhound/products/DCO/ticket/303.

**Pull request merged**